salt/metalk8s/orchestrate/solutions/init.sls

```diff
@@ -0,0 +1,158 @@
+{% set version = pillar.metalk8s.nodes[pillar.bootstrap_id].version %}
+{% set kubeconfig = "/etc/kubernetes/admin.conf" %}
+{% set context = "kubernetes-admin@kubernetes" %}
+
+# Init
+Make sure solutions namespace exist:
+  metalk8s_kubernetes.namespace_present:
+    - name: metalk8s-solutions
+    - kubeconfig: {{ kubeconfig }}
+    - context: {{ context }}
+
+Make sure UIs namespace exist:
+  metalk8s_kubernetes.namespace_present:
+    - name: admin-uis
```

Comment:
Now that I think of it, since they are all in the same Namespace, we can use `metalk8s-solutions` for the UIs and ConfigMaps IMO.

Reply:
isnt it hardcoded in the deployment.yaml of the ui ?